### PR TITLE
feat(RHINENG-8714): add resource string and change app string

### PR DESCRIPTION
**Pull request by jpramos123**
# Overview

This PR is being created to address [RHINENG-8714](https://issues.redhat.com/browse/RHINENG-8714).

Add resource string, change the app string and simplify the validation logic.

## PR Checklist

- [ ] Keep PR title short, ideally under 72 characters
- [ ] Descriptive comments provided in complex code blocks
- [ ] Tests: validate optimal/expected output
- [ ] Tests: validate exceptions and failure scenarios
- [ ] Tests: edge cases
- [ ] Recovers or fails gracefully during potential resource outages (e.g. DB, Kafka)
- [ ] Uses [type hinting](https://docs.python.org/3/library/typing.html), if convenient
- [ ] Documentation, if this PR changes the way other services interact with host inventory
- [ ] Links to related PRs

## Secure Coding Practices Documentation Reference

You can find documentation on this checklist [here](https://github.com/RedHatInsights/secure-coding-checklist).

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Review by kruai — CHANGES_REQUESTED** [review]

File: app/queue/queue.py [hunk 1]
```diff
@@ -457,7 +444,7 @@ def export_service_event_loop(consumer, flask_app, interrupt):
                     logger.error(f"Message received but has an error, which is {str(msg.error())}")
                     metrics.ingress_message_handler_failure.inc()
                 else:
-                    logger.debug("Message received")
+                    logger.debug("Export Service message received")
                     try:
                         handle_export_message(msg.value())
                         metrics.consumed_message_size.observe(len(str(msg).encode("utf-8")))
```
[the remark this answers is not in the document]
Comment (jpramos123) [reply app/queue/queue.py:450]: Thanks, removed it!

File: app/queue/export_service.py [hunk 2]
```diff
@@ -8,6 +8,8 @@
 
 logger = get_logger(__name__)
 
+EXPORT_SERVICE_RESOURCE = "urn:redhat:application:inventory:export:systems"
+
```
[the remark this answers is not in the document]
Comment (kruai) [reply app/queue/export_service.py]: OK. Still might be better for it to just be `export:systems` appended to `EXPORT_SERVICE_APPLICATION`, but it's nothing to block on.

**Review by kruai — APPROVED** [review]

**Pull request merged**